extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/model/TransferRequestDto.java

```diff
@@ -29,7 +29,6 @@ public class TransferRequestDto {
 
     @NotNull
     private String connectorAddress;
-    @NotNull
     private String id;
```

Comment:
Why was this removed? Transfer requests should/must always have an ID

Reply:
@paullatzelsperger I think this was a typo, as every system-test doesn't pass an id in the request as the EDC will generate one by itself in the transformer:

https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/60da937f68c72f471419c2c779ac1d499bc0517b/extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/transform/TransferRequestDtoToDataRequestTransformer.java#L46-L47